TypingPall/Views/Editor/TypingEditor.swift

```diff
@@ -6,6 +6,8 @@ struct TypingEditor: NSViewRepresentable {
     @Binding var placeholder: String
     var fontSize: CGFloat
 
+    @AppStorage("tabEqualsToSpaces") var spaces: Double = 4
```

Comment:
will use strongly typed AppStorage key in future https://gist.github.com/DivineDominion/3060eaceb6f2f9b65b6d183ac8b8dba9